### PR TITLE
Add ForwardOrderConfirmationMessage

### PR DESCRIPTION
**Pull request by CarsonBradshaw**

### CODE REVIEW

File: messages/src/main/java/messages/ForwardOrderConfirmationMessage.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    public enum Status {
+        SUCCESS,
+        ERROR,
+        ;
+
+        public byte toByte() {
+            return (byte) this.ordinal();
+        }
+
+        public static Status getStatusFromByte(byte orderTypeByte) {
```
Comment (Kardbord) [inline messages/src/main/java/messages/ForwardOrderConfirmationMessage.java]:
```suggestion
        public static Status getStatusFromByte(byte statusByte) {
```

File: messages/src/main/java/messages/ForwardOrderConfirmationMessage.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            return Status.values()[orderTypeByte];
```
Comment (Kardbord) [inline messages/src/main/java/messages/ForwardOrderConfirmationMessage.java]:
```suggestion
            return Status.values()[statusByte];
```

Comment (CarsonBradshaw) [reply messages/src/main/java/messages/ForwardOrderConfirmationMessage.java]: Done.

Comment (CarsonBradshaw) [reply messages/src/main/java/messages/ForwardOrderConfirmationMessage.java]: Jk.

Comment (TannerKvarfordt) [reply messages/src/main/java/messages/ForwardOrderConfirmationMessage.java]: We'll want to change it in both places. :stuck_out_tongue_winking_eye:

Comment (CarsonBradshaw) [reply messages/src/main/java/messages/ForwardOrderConfirmationMessage.java]: Done for real this time.

**Comment by CarsonBradshaw** [comment]
@Palexander21 No big deal on this one but here's an example where Squash and Merge would have been preferred to Rebase.

Since each commit in this PR was related to the same change, they could have been in a single commit (hence the Squash). This way master doesn't get a bunch of tiny commits.

**Pull request merged**